crossfit/backend/torch/loader.py

```diff
@@ -66,7 +75,11 @@ def __next__(self):
 
         batch = {key: val[self.current_idx : end] for key, val in self.tensor_dict.items()}
         if self.max_seq_len is not None:
-            batch = {key: val[:, : self.max_seq_len] for key, val in batch.items()}
+            # TODO: Check this
```

Comment:
Not sure if the right padding has been checked, but the left padding seems to work fine.

Reply:
Forgot to remove that TODO:

Reply:
It's still there if it matters to you.